.travis.yml

```diff
@@ -7,8 +7,8 @@ go:
 env:
   global:
     - PATH=${PATH}:./bin
-    - POP_PATH=$PWD/cmd/proxy
-    - GO_ENV=test_postgres
+    - POP_PATH=$PWD/cmd/olympus
```

Comment:
do we need POP_PATH at all?

Reply:
yes! because we still have the CDN rdbms driver which is dependent on Pop. So, I made sure that the proxy is independent of `Pop` but `olympus` is not.
Once, we have voted on CDN to be kicked out of olympus, the next step would be to create a PR that basically removes `POP_PATH` from travis

Reply:
@manugupt1 so say we _keep_ CDN, do we still need RDBMS in the CDN? can we not remove RDBMS and use another storage for CDN?

Reply:
You are right on!

So, if we decide not to support the RDBMS driver for CDN, we do not need it at all.
For example,
https://github.com/gomods/athens/blob/master/pkg/cdn/metadata/cdn_metadata_entry.go#L14-L19

Supports both mongo and SQL based tags!
If we remove the rdbms driver that stores the metadata https://github.com/gomods/athens/tree/master/pkg/cdn/metadata/rdbms

Then, we can remove all the dependency.

Reply:
I believe we decided to remove RDBMS from anything Athens related, but we can do it in another PR. @arschles correct me if im wrong :)

Reply:
Oh cool! I was not aware of of that! Let us wait for @arschles to confirm.

Reply:
I do believe that RDBMS in CDN is without use. there are no relationships there so any document db will do just fine. Let's get rid of it

Reply:
Feel free to address it in this or a separate PR @manugupt1